editor.planx.uk/src/@planx/components/Section/Public.tsx

```diff
@@ -153,71 +153,112 @@ export function SectionsOverviewList({
-            />
-          </SectionTitle>
-          <SectionState>
-            {getTag(sectionStatuses[sectionId], sectionNode.data.title)}
-          </SectionState>
-        </SectionRow>
-      ))}
+      {Object.entries(sectionNodes).map(([sectionId, sectionNode]) => {
+        const status = sectionStatuses[sectionId];
+        const clickHandler = getClickHandler(sectionId, status);
+        const isClickable = clickHandler !== undefined;
+        const statusId = `section-${sectionId}-status`;
+        const hintId = `section-${sectionId}-hint`;
+
+        return (
+          <SectionRowWrapper
+            key={sectionId}
+            isClickable={isClickable}
+            onClick={
+              isClickable
+                ? (e) => {
+                    if (
+                      e.target === e.currentTarget ||
+                      !e.currentTarget
+                        .querySelector("a")
+                        ?.contains(e.target as Node)
+                    ) {
+                      clickHandler?.();
+                    }
+                  }
+                : undefined
+            }
+          >
+            <SectionContent>
+              <SectionDescription>
+                {isClickable ? (
+                  <SectionTitleLink variant="subtitle1" component="h4">
+                    <Link
+                      href="#"
+                      onClick={(e) => {
+                        e.preventDefault();
+                        clickHandler?.();
+                      }}
+                      aria-describedby={
+                        sectionNode.data.description
+                          ? `${hintId} ${statusId}`
+                          : statusId
+                      }
+                    >
+                      {showChange && status === SectionStatusEnum.Completed && (
+                        <span style={visuallyHidden}>{`Change `}</span>
+                      )}
+                      {sectionNode.data.title}
+                    </Link>
+                  </SectionTitleLink>
+                ) : (
+                  <SectionTitleText variant="subtitle1" component="h4">
+                    {sectionNode.data.title}
+                  </SectionTitleText>
+                )}
+                {sectionNode.data.description && (
+                  <SectionHint id={hintId}>
+                    <ReactMarkdownOrHtml
+                      source={sectionNode.data.description}
+                      openLinksOnNewTab
+                    />
+                  </SectionHint>
+                )}
+              </SectionDescription>
+              <SectionStatus id={statusId}>
+                {getTag(status, sectionNode.data.title)}
+              </SectionStatus>
+            </SectionContent>
+          </SectionRowWrapper>
+        );
+      })}
     </DescriptionList>
   );
 }
 
-const Table = styled("dl")(({ theme }) => ({
+const Table = styled("ul")(({ theme }) => ({
```

Comment:
It's probably worth renaming some of these components - `Table` is not that meaningful now it's not a `dl` (arguably it wasn't a great name to begin with!). The same applies the the `DescriptionList` component.